pkg/server/registry/apigroups/acorn/volumes/translator.go

```diff
@@ -22,6 +22,33 @@ type Translator struct {
 }
 
 func (t *Translator) FromPublicName(ctx context.Context, namespace, name string) (string, string, error) {
+
+	i := strings.LastIndex(name, ".")
```

Comment:
Would there be situations where there would be more than one `.`? If no, can we use `strings.Cut` as similarly used in other places? Just for consistency.

Reply:
Or if the goal is to just see whether the name contains a `.` at all, we could use `strings.Contains`.

Reply:
I copied the same approach Darren used in secrets/translator.go:

```
func (t *Translator) FromPublicName(ctx context.Context, namespace, name string) (string, string, error) {
	i := strings.LastIndex(name, ".")
	if i == -1 || i+1 > len(name) {
		return namespace, name, nil
	}

	prefix := name[:i]
	secretName := name[i+1:]
```

@ibuildthecloud is there some logic I'm missing here. For both this comment and @g-linville's comment below: